server/models/data/availabilitydetails.js

```diff
@@ -0,0 +1,19 @@
+const moment = require('moment');
+
+class AvailabilityDetails {
+
+    constructor(availabilityData) {
+        this.schoolId = availabilityData.schoolId_id;
+        // At any given time, either the worker or student will be null.
+       // this.worker = availabilityData.worker_id,
```

Comment:
Hi Nancy,
I would recommend removing the commented lines or providing a brief explanation as to why they are kept to prevent future confusion.

Reply:
Hi Amy,
Thank you for suggesting that!
My action: I've removed the comments on line 10,11,12 and 15. The constructor `availabilityData` only contains the following parameters: `workerId`, `schoolId`, 'startTime`, and `endTime`.

Please confirm if this takes care of your suggestion!